third_party/cxxopts/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2014 Jarryd Beck
```

Comment:
Why isn't this a submodule?

Reply:
https://github.com/jarro2783/cxxopts includes a bit more than just header and license. I don't really see an advantage for submodules here (we currently include JSON the same way), but that's your call, you decide.

Reply:
Since that is a project that is quite active, I think that having a submodule makes more sense because it makes it easier to pull updates. Not sure why that didn't happen with json.

Reply:
oh, that would have been my fault as well, with the same reasoning ;)

Reply:
Can you specify which files to check out perhaps? Or is it all or nothing?

Reply:
Nope, but I don't see a problem with having a few kb in the third_party folder